### ISSUE TITLE
Token feature, or something like that.

### ISSUE THREAD

**Issue by kant2002** [WICG/open-ui#226]
I lurking around SAP Fiori design guide and found one interesting concept which is probably missing from regular HTML talks, but very valuable concept in the productive applications.

In SAP this is called [token](https://experience.sap.com/fiori-design-web/token/).

![image](https://user-images.githubusercontent.com/4257079/100539375-28f00580-3260-11eb-9bb4-003d857c15fb.png)

These ussually applied when used inside some combobox/dropdown concept.
Does this building block warranted to be discussed?

**Comment by una** [comment]
Is this similar to [chip](https://material.io/components/chips)?

**Comment by kant2002** [comment]
Yes. In the same spirit. What is confusing me is that it is used as part of other components mostly. Somebody mix that with Badge concept

**Comment by una** [comment]
I do believe these are different from badges. Chips/tokens are usually used in a filtering mechanism, and are interactive, whereas badges tend to convey information and be static.

**Comment by gregwhitworth** [comment]
@kant2002 I would love to see this one progress as I have a feeling I can foresee a composite control to @una point of this being how the values are represented in a combobox. I'd love to see a proposal for this if it is common across component libs

**Comment by brianespinosa** [comment]
I was just thinking about what these should officially called yesterday as I have seen them called different things across libs. Semantic UI calls them [Labels](https://semantic-ui.com/elements/label.html) (obvious semantics problem with HTML `<label>`).

**Comment by bkardell** [comment]
I will just note that aria has since closed their related issue on this pushing it back to  HTML should have it before ARIA... I'm not honestly sure how I feel about that, but do we want to pursue this still? If so I think it needs a champion.

**Comment by keithamus** [comment]
As per the meeting logs, we reviewed this and while we're open to the idea of adding some kind of chip or token element, no one is currently able to champion such a proposal.

We'll close this as not planned, but if anyone is interested in writing a proposal and presenting it to the group then we welcome that, and we can re-open the issue. Championing a proposal is quite straightforward: you need to be someone who wants the feature in the platform, and is willing to write a document to explain how that might work. The OpenUI team can work with you to make it a reality.
